test/integration/setup.js

```diff
@@ -94,16 +94,21 @@ const initialize = async () => {
   return withDefaults({ db, bcrypt, context }).transacting(populate);
 };
 
-before(initialize);
+// eslint-disable-next-line func-names, space-before-function-paren
```

Comment:
I've toned down `space-before-function-paren` in Frontend, I could do something similar in Backend if that's of interest.